recipe/meta.yaml

```
@@ -10,27 +10,25 @@ package:
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: c93fc0ed8216c71f9b8264788ca7743e26f4498b2011e5e2404696e5fd238e68
+  sha256: c8e9d769a6066bb5f94bb0362dbc20ed6f6b525336373263aeb6f9c6dc7b91e4
 
 build:
   number: 0
-  skip: true  # [py<37 or osx]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<37 or osx or cuda_compiler_version != "11.7"]
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps -vv
```

Forgot about this... 😅 We need to ignore the `run_exports` coming from the nvcc compiler:
```suggestion
  script:
    - {{ PYTHON }} -m pip install . --no-deps -vv
  ignore_run_exports:
    - cudatoolkit
```